web/src/components/core/Section.jsx

```diff
@@ -97,7 +100,7 @@ export default function Section({
       <Header />
       <div className="stack content">
         {errors?.length > 0 &&
-          <ValidationErrors errors={errors} title={`${title} errors`} />}
+          <ValidationErrors errors={errors} sectionName={title} />}
```

Comment:
What about using `sectionId` instead? BTW, this made me realize that maybe sections should have an `id` too, but that's could be something for #892